src/RestApi/StoreApi/Utilities/CartController.php

```diff
@@ -52,21 +54,44 @@ public function add_to_cart( $request ) {
 				$variation_id = 0;
 			}
 
-			$cart_id          = wc()->cart->generate_cart_id( $product_id, $variation_id, $request['variation'], $request['cart_item_data'] );
+			$cart_id          = wc()->cart->generate_cart_id(
+				$product_id,
+				$variation_id,
+				$request['variation'],
+				$request['cart_item_data']
+			);
 			$existing_cart_id = wc()->cart->find_product_in_cart( $cart_id );
 
 			if ( ! $product->is_purchasable() ) {
-				throw new RestException( 'woocommerce_rest_cart_product_is_not_purchasable', __( 'This product cannot be purchased.', 'woo-gutenberg-products-block' ), 403 );
+				throw new RestException(
+					'woocommerce_rest_cart_product_is_not_purchasable',
+					__( 'This product cannot be purchased.', 'woo-gutenberg-products-block' ),
+					403
+				);
 			}
 
 			if ( $product->is_sold_individually() && $existing_cart_id ) {
-				/* translators: %s: product name */
-				throw new RestException( 'woocommerce_rest_cart_product_sold_individually', sprintf( __( '"%s" is already inside your cart.', 'woo-gutenberg-products-block' ), $product->get_name() ), 403 );
+				throw new RestException(
+					'woocommerce_rest_cart_product_sold_individually',
+					sprintf(
+						/* translators: %s: product name */
+						__( '"%s" is already inside your cart.', 'woo-gutenberg-products-block' ),
+						$product->get_name()
+					),
+					403
+				);
 			}
 
 			if ( ! $product->is_in_stock() ) {
-				/* translators: %s: product name */
-				throw new RestException( 'woocommerce_rest_cart_product_no_stock', sprintf( __( 'You cannot add &quot;%s&quot; to the cart because the product is out of stock.', 'woo-gutenberg-products-block' ), $product->get_name() ), 403 );
+				throw new RestException(
+					'woocommerce_rest_cart_product_no_stock',
+					sprintf(
+						/* translators: %s: product name */
+						__( 'You cannot add &quot;%s&quot; to the cart because the product is out of stock.', 'woo-gutenberg-products-block' ),
```

Comment:
should `&quot;` be moved to the sprintf replacement instead of embedded in the string (to prevent potential breakage by translators?)

Reply:
No I don't think so. That character may need translation too. Different languages use different punctuation.